scripts/list_materials.py

```
@@ -0,0 +1,280 @@
+            continue
+
+        # key: value または key:
+        m = re.match(r'^([A-Za-z_][A-Za-z0-9_]*)\s*:(.*)', line)
+        if not m:
+            i += 1
+            continue
+
+        key = m.group(1)
+        raw_val = m.group(2).strip()
+
+        # コメント除去（ # 以降）
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)